examples/main.tf

```diff
@@ -21,9 +27,8 @@ module "kms_key" {
 }
 
 module "cluster" {
-  source  = "boldlink/ecs-cluster/aws"
-  version = "1.0.0"
-  name    = local.name
+  source = "./../"
+  name   = local.name
   configuration = {
     execute_command_configuration = {
       kms_key_id = try(module.kms_key[0].key_id, null)
```

Comment:
I have a question here. Is it possible to see the ECS associated kms key from the console?

Reply:
Yes Patrick, it is possible...by adding kms to the outputs section. for there to be an output create kms_key option should be turned on in the locals as well. I have added these options in the most recent push.

Reply:
Okay